mix.exs

```diff
@@ -40,6 +40,7 @@ defmodule Phoenix.Mixfile do
 
      # Test dependencies
      {:phoenix_html, "~> 1.2", only: :test},
+     {:ecto, "~> 0.14.1", only: :test},
```

Comment:
Yeah, we should not depend on Ecto. Please just export the __schema__(:source) function from a blank module.